www/sw.min.js

```diff
@@ -1 +1 @@
eFilterSimple[i].methods[o].parameters.findIndex(function(t){return t.keyValues.length==e.data.data.cacheIfParametersMatch.length&&e.data.data.cacheIfParametersMatch.every(function(e){return 2==e.length&&-1!=t.keyValues.findIndex(function(t){return n(t.key,e[0])&&n(t.value,e[1])})})}))?requestsCacheFilterSimple[i].methods[o].parameters.push({keyValues:e.data.data.cacheIfParametersMatch.map(function(e){return{key:e[0],value:e[1]}}),ignoreKeys:e.data.data.ignoreParameterKeysOnCache}):requestsCacheFilterSimple[i].methods[o].parameters[u]={keyValues:e.data.data.cacheIfParametersMatch.map(function(e){return{key:e[0],value:e[1]}}),ignoreKeys:e.data.data.ignoreParameterKeysOnCache},r({status:"OK"})}else if("requests_skip_filter"==e.data.action){var i,o,u;e.data.data.url=urlAddBaseLocation(e.data.data.url).toLowerCase(),e.data.data.method=e.data.data.method.toLowerCase(),e.data.data.header=e.data.data.header.map(function(e){return e.toLowerCase()}),2==e.data.data.header.length?e.data.data.header={key:e.data.data.header[0],value:e.data.data.header[1]}:e.data.data.header={key:"",value:""},-1==(i=requestsSkipOnFetch.findIndex(function(t){return n(e.data.data.url,t.url)}))&&(requestsSkipOnFetch.push({url:s(e.data.data.url),methods:[]}),i=requestsSkipOnFetch.length-1),-1==(o=requestsSkipOnFetch[i].methods.findIndex(function(t){return n(e.data.data.method,t.method)}))&&(requestsSkipOnFetch[i].methods.push({method:s(e.data.data.method),headers:[]}),o=requestsSkipOnFetch[i].methods.length-1),-1==(u=requestsSkipOnFetch[i].methods[o].headers.findIndex(function(t){return n(t.key,e.data.data.header.key)&&n(t.value,e.data.data.header.value)}))&&requestsSkipOnFetch[i].methods[o].headers.push(e.data.data.header),r({status:"OK"})}else"load_and_cache_urls"==e.data.action?a(Promise.all(e.data.data.urls.map(function(t){return function(t,r){var a=new Request(urlAddBaseLocation(t));return fetch(a).then(function(n){if(200!=n.status||"basic"!=n.type)return!1;var s=new Request(filterUrlParameters(a.url,r.map(function(e){return e.toLowerCase()})));caches.open(CACHE_NAME_DYNAMIC).then(function(r){return r.put(s,n.clone()),sendMessageToClient(e,{msg:"Cached resource:",url:t,urlCached:s.url}),!0}).catch(function(){return!1})}).catch(function(){return!1})}(t,e.data.data.ignoreParameterKeysOnCache)}))):"check_urls_in_cache"==e.data.action?(t=e.data.data.urls,Promise.all(t.map(function(e){return caches.match(urlAddBaseLocation(e),{ignoreSearch:!1}).then(function(t){return t?e:""}).catch(function(){return""})})).then(function(e){return e.filter(function(e){return""!=e})}).then(function(e){return{urls:e,status:"OK"}}).catch(function(){return{urls:[],status:"Failed"}})).then(r):"get_service_worker_version"==e.data.action?r({data:SERVICE_WORKER_VERSION}):"set_use_cache_only_in_offline"==e.data.action?(CacheMode.UseOnlyCacheInOffline=e.data.enabled,r({status:"OK"})):"get_requests_stats"==e.data.action?r({data:requestsCount}):r({status:"Failed",error:"Unknown operation: "+e.data.action})});
```

Comment:
Is this file still in use? Our applications use their own `sw.<app>.min.js`

Reply:
It is minified version of `base` sw file. Applications, which are generated by meta_app, it does not use it now. But may be it usefull for other applications.